src/intelliSense/configCompletionItemProvider.ts

```diff
@@ -100,6 +100,14 @@ export class ConfigCompletionItemProvider implements vscode.CompletionItemProvid
             completionItem.range = overwriteRange;
             completionItem.insertText = value + separator;
             completionItem.kind = vscode.CompletionItemKind.Value;
+
+            // Deprioritize image placeholders with platform (e.g., ${MODULES.SampleModule.amd64}) in completion item list
```

Comment:
what I am thinking is not deprioritize, but the correct group together of names. Can we use the value without "${}" for sort?

Reply:
It's already grouped (master branch):
![image](https://user-images.githubusercontent.com/716650/48176563-9401d300-e34b-11e8-95d4-fb9fc99b0d1c.png)

Do you mean we also need to remove  platforms like amd64 from sortText?

Reply:
I mean sort the content in ${ }, MODULES.SampleFunction... Instead of the whole string  ${MODULES.SampleFunction....}  . use "split by ." is not robust. Since user may have folder name containing  "."